plottr/plot/mpl.py

```diff
@@ -169,8 +169,12 @@ def __call__(self, value: float, clip: Optional[bool] = None) -> np.ma.core.Mask
         return super().__call__(value, clip)
 
 
-def setMplDefaults() -> None:
+def setMplDefaults(obj: Any) -> None:
     """Set some reasonable matplotlib defaults for appearance."""
+    if isinstance(obj, _MPLPlotWidget):
+        fontSize = 6 * obj.devicePixelRatio()
+    else:
+        fontSize = 6
```

Comment:
Can you explain why this can only apply to _MPLPlotWidget ?

Reply:
These are global matplotlib settings so they will apply to all matplotlib figures independently of the window they are set on

Reply:
I suggest that we apply this all the time and change the type of object from `Any` to `QtWidgets.QWidget`